src/Propulsion.MessageDb/ReaderCheckpoint.fs

```diff
@@ -0,0 +1,81 @@
+module Propulsion.MessageDb.ReaderCheckpoint
+
+open Npgsql
+open NpgsqlTypes
+open Propulsion.Feed
+open Propulsion.Infrastructure
+open System
+
+let createIfNotExists (conn : NpgsqlConnection, schema: string) =
+    let cmd = conn.CreateCommand()
+    cmd.CommandText <- $"
+      create table if not exists {schema}.propulsion_checkpoints (
```

Comment:
if the convention in mdb is to pluralise the table name, I guess this is more debatable, but I personally prefer singular

Reply:
The main table is `messages` so it is pluralized. But you can create this checkpoint table in any schema so happy to have it singular

Reply:
also it might make sense to wire the init into `propulsion init` (IIRC that can be used to create the SSS checpoint store)

Reply:
Added a note